packages/tailwindcss/tests/ui.spec.ts

```diff
@@ -317,6 +316,7 @@ async function render(page: Page, content: string) {
 
   await page.setContent(content)
 
+  let scanner = new Scanner({})
```

Comment:
fyi @RobinMalfait I fixed the UI tests — scanner being stateful caused problems because scanFiles only returns new candidates now. I think using a new scanner every time here is probably a good idea regardless.